### ISSUE TITLE
Implement Streamer Mode

### ISSUE THREAD

**Issue by big213** [Kubiverse/Cube-Zone#40]
room setting

show scramble when timing
show scoreboard

**Comment by big213** [comment]
results table could start counting up from 0 after "solving" state. due to latency, would be approximation of the solve time.

**Comment by Odder** [comment]
Yeah the latency between the status changing to "solving" and to "validating" should be equivalent, so you should be fairly safe to assume that solving -> starting timer and validating -> stopping timer (at which point you actually DO have the time available)

**Comment by big213** [comment]
Streamer room setting for width of the central scoreboard
